Packages/Tracking/Core/Runtime/Scripts/Hands/CapsuleHand.cs

```diff
@@ -380,6 +380,11 @@ public override void UpdateHand()
                     else
                     {
                         position = finger.GetBone((Bone.BoneType)j).NextJoint;
+                        if (j == 3)
+                        {
+                            // Fingertips need to be adjusted back by a radii to account for sphere drawing.
+                            position -= finger.GetBone((Bone.BoneType)j).Direction * _jointRadius;
```

Comment:
IIRC the size of the capsule sphere can be set in the editor and is therefore not necessarily derived from the joint radius data?

Reply:
`_jointRadius` is the editor controlled field for the drawn sphere size. Because the data coming from LeapC or from OpenXR and converted to LeapC format has this datapoint _on_ the surface, we actually just need to adjust back by the drawn radius, not the actual radius.